src/Makefile.am

```diff
@@ -149,7 +149,7 @@ endif
 core-daemons: ceph-mon ceph-osd ceph-mds radosgw
 admin-tools: monmaptool osdmaptool crushtool ceph-authtool
 base: core-daemons admin-tools \
-	cephfs ceph-syn ceph-conf \
+	cephfs ceph-fuse ceph-syn ceph-conf \
```

Comment:
I can't tell what this "base" target is for, but I wonder if we also have to fix something in CMake.